src/components/Explore/utils.ts

```diff
@@ -303,9 +317,10 @@ export function getExportImageUrl(sharedComponentId: string) {
   return urlJoin(share_image_url, `share/${sharedComponentId}/export.png`);
 }
 
-export function getChartUrl(fips: string, sharedComponentId: string) {
+export function getChartUrl(sharedComponentId: string) {
+  const pathname = window.location.pathname;
   const redirectTo = urlJoin(
-    getRelativeUrlForFips(fips),
+    pathname.includes('us/') ? pathname : '/',
```

Comment:
This is a little fragile / broken because the pathname may have additional stuff at the end (/compare/x, /chart/x, /explore/x, etc.)

As a simple example if I share an explore chart, I might end up with a URL like:
https://covid-projections-git-pablo-explore-homepage-share.covidactnow.vercel.app/us/north_dakota-nd/explore/149

If I go there and share it again, I end up at a broken URL:
https://covid-projections-git-pablo-explore-homepage-share.covidactnow.vercel.app/us/north_dakota-nd/explore/149/explore/150

Because /explore ends up appended twice....